min_dom_set_z3.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+import pygraphviz as pgv
+from z3 import *
+import sys
+
+def min_dom_set(graph):
+    """Try to dominate the graph with the least number of verticies possible"""
+    s = Optimize()
```

Comment:
Awesome! I didn't even know this feature - maybe I'll see if it's not worth making a simple example for the essentials directory :+1: